agent-simulator/tomcatconf/commands-simulator.properties.in

```diff
@@ -1,21 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@borisroman I've removed this file as part of this PR. Can you confirm it relates tot he simulator hypervisor plugin?